res/controllers/Keyboard-Default.kbd.xml

```diff
@@ -0,0 +1,382 @@
+<?xml version="1.0" ?>
+<MixxxKeyboardPreset mixxxVersion="2.0.1+" schemaVersion="1">
+  <info>
+    <name>Default Mapping</name>
+    <author>Mixxx</author>
+    <description>Default mapping for Mixxx.</description>
+  </info>
+  <controller>
+    <group name="[AutoDJ]">
+      <control action="enabled">
+        <keyseq final="1">Shift+F12</keyseq>
+      </control>
+      <control action="fade_now">
+        <keyseq final="1">Shift+F11</keyseq>
+      </control>
+      <control action="shuffle_playlist">
+        <keyseq final="1">Shift+F9</keyseq>
+      </control>
+      <control action="skip_next">
+        <keyseq final="1">Shift+F10</keyseq>
+      </control>
+    </group>
+    <group name="[Channel1]">
+      <control action="LoadSelectedTrack">
+        <keyseq final="1">Shift+Left</keyseq>
+      </control>
+      <control action="back">
+        <keyseq lang="en_US" scancode="30">a</keyseq>
```

Comment:
Yes, that would be a correct schema. However, it looks like it exposes too much internals to the user. Do we need the scancode option? 

```XML
<control action="some_action">
  
  <!-- Binds to "Shift+A" key on all not explicit listed layouts. 
  So, this won't work for layouts that lack a key labeled 'A'. -->
  <keyseq>"Shift+A"</keyseq>

  <!-- Binds to "Shift+A" on a "en_US" layout only-->
  <keyseq lang="en_US">"Shift+A"</keyseq>

  <!-- Binds to Shift+[key_with_scancode_30] to all not explicit listed layouts -->
 <keyseq byPositionOf="en_US">"Shift+A"</keyseq>  
```